app/src/main/java/com/bogdan/codeforceswatcher/features/news/NewsFragment.kt

```diff
@@ -36,9 +84,27 @@ class NewsFragment : Fragment(), SwipeRefreshLayout.OnRefreshListener, StoreSubs
 
     override fun onStop() {
         super.onStop()
+
         store.unsubscribe(this)
     }
 
+    private fun onRefresh() {
+        store.dispatch(NewsRequests.FetchNews(true))
```

Comment:
It's always better to include the argument name in a call when you have a literal value passed (`true`, `false`, `42`, `hello`, etc.).